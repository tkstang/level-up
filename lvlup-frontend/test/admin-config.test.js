import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import AdminConfiguration from '../src/components/Admin/config/config-main';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('AdminConfiguration should render', () => {
  const store = mockStore({ lvlupApp });
  const config = shallow(
    <AdminConfiguration store={store} />,
  );
  expect(shallowToJson(config)).toMatchSnapshot();
});
