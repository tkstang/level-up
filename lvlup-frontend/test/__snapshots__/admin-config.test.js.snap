exports[`test AdminConfiguration should render 1`] = `
<Container>
  <Segment
    className="cohortOptions"
    textAlign="center">
    <h2>
      Cohort Options
    </h2>
    <Link
      replace={false}
      to="/admin/configuration/addcohort">
      <Button
        as="button"
        color="orange"
        inverted={true}>
         Click here to add a Cohort
      </Button>
    </Link>
    <br />
    <br />
    <Link
      replace={false}
      to="/admin/configuration/addadmincohort">
      <Button
        as="button"
        color="orange"
        inverted={true}>
         Click here to add a Cohort to your Profile
      </Button>
    </Link>
  </Segment>
</Container>
`;
